tests/unittests/CoreGraphics.Drawing/CGContextDrawing_ShadowTests.cpp

```diff
@@ -50,3 +50,53 @@ DISABLED_DRAW_TEST_F(CGContext, ShadowWithRotatedCTM, WhiteBackgroundTest<>) {
 
     CGContextStrokeRect(context, rect);
 }
+
+DISABLED_DRAW_TEST_F(CGContext, ShadowOverlap, WhiteBackgroundTest<>) {
```

Comment:
I am not sure if this is how we want it.  For draw tests, our plan is to document the differences with reference platform and then replace the reference images with the baseline winobjc images.  This way, the tests will catch future regressions.  @DHowett-MSFT?